doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.makeWarp.MakeWarpTask.rst

```diff
@@ -0,0 +1,168 @@
+.. lsst-task-topic:: lsst.pipe.tasks.makeWarp.MakeWarpTask
+
+############
+MakeWarpTask
+############
+
+Warp and optionally PSF-Match calexps onto a common projection
+
+Warp and optionally PSF-Match calexps onto a common projection by
+performing the following operations:
+
+- Group calexps by visit/run
+- For each visit, generate a Warp by calling method @ref run.
+
+`run` loops over the visit's calexps calling
+`~lsst.pipe.tasks.warpAndPsfMatch.WarpAndPsfMatchTask` on each visit
+
+
+WarpType identifies the types of convolutions applied to Warps
+(previously CoaddTempExps). Only two types are available: direct
+(for regular Warps/Coadds) and psfMatched (for Warps/Coadds with
+homogenized PSFs). We expect to add a third type, likelihood, for
+generating likelihood Coadds with Warps that have been correlated with
+their own PSF.
+
+To make `psfMatchedWarps`, select `config.makePsfMatched=True`. The subtask
+`~lsst.ip.diffim.modelPsfMatch.ModelPsfMatchTask`
+is responsible for the PSF-Matching, and its config is accessed via
+`config.warpAndPsfMatch.psfMatch`.
+
+The optimal configuration depends on aspects of dataset: the pixel scale,
+average PSF FWHM and dimensions of the PSF kernel. These configs include
+the requested model PSF, the matching kernel size, padding of the science
+PSF thumbnail and spatial sampling frequency of the PSF.
+
+Processing summary
+==================
+
+ToDo
```

Comment:
Please file the ticket to do this work, note it here, and link it as "relates to" on DM-37673.

Reply:
Ticket [DM-38869](https://jira.lsstcorp.org/browse/DM-38869) has been created for this future work.